### PR TITLE
Fixes some redirects + pwa support

### PR DESCRIPTION
**Pull request by carsonfarmer**
Signed-off-by: Carson Farmer <carson.farmer@gmail.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,7 +11,7 @@
 # production
 /build
 /staging
-/prod
```
Comment (carsonfarmer) [inline .gitignore:14]: i'm using these folders to manage our buckets, so i don't wanna accidentally push them :P

File: public/manifest.json [hunk 2]
```diff
@@ -18,8 +18,8 @@
       "sizes": "512x512"
     }
   ],
-  "start_url": ".",
+  "start_url": "/",
```
Comment (carsonfarmer) [inline public/manifest.json:21]: it looks like pwa's need this, but if not, we can switch back

File: src/containers/SignUpForm.tsx [hunk 3]
```diff
@@ -27,7 +27,7 @@ const SignUpForm = () => {
                 return;
               }
             }
-            history.push("/success");
+            history.push("../success");
```
Comment (carsonfarmer) [inline src/containers/SignUpForm.tsx:30]: use relative links, but that might be wrong... might need to be . now that i'm seeing this

File: src/index.tsx [hunk 4]
```diff
@@ -5,6 +5,7 @@ import { CookiesProvider } from "react-cookie";
 import "./index.css";
 import App from "./App";
 import reportWebVitals from "./reportWebVitals";
+import * as serviceWorkerRegistration from "./serviceWorkerRegistration";
```
Comment (carsonfarmer) [inline src/index.tsx:8]: service worker support for pwas

File: src/index.tsx [hunk 5]
```diff
@@ -24,6 +25,11 @@ ReactDOM.render(
   document.getElementById("root")
 );
 
+// If you want your app to work offline and load faster, you can change
+// unregister() to register() below. Note this comes with some pitfalls.
+// Learn more about service workers: https://cra.link/PWA
+serviceWorkerRegistration.register();
```
Comment (carsonfarmer) [inline src/index.tsx:31]: we register our app here

File: src/pages/AccessPage.tsx [hunk 6]
```diff
@@ -70,24 +71,24 @@ const BackgroundContainer = styled.div`
   right: 50%;
 `;
 
-const PrivateRoute = ({
```
Comment (carsonfarmer) [inline src/pages/AccessPage.tsx:73]: some temp stuff to work around redirect issues for now

File: src/pages/AccessPage.tsx [hunk 7]
```diff
@@ -139,8 +140,10 @@ const AccessPage = () => {
         <div className="content">
           <TextileLogo />
           <Switch>
-            <Route path="/signup" component={SignUpForm} />
-            <PrivateRoute path="/" component={SuccessForm} />
+            <Route path="/success" component={SuccessForm} />
```
Comment (carsonfarmer) [inline src/pages/AccessPage.tsx:143]: we're not using private routes for now, and we'll redirect everything except success to the base url for now

File: src/service-worker.ts [hunk 8]
```diff
@@ -0,0 +1,81 @@
+/// <reference lib="webworker" />
```
Comment (carsonfarmer) [inline src/service-worker.ts:1]: service working stuff, copies from cra

File: src/serviceWorkerRegistration.ts [hunk 9]
```diff
@@ -0,0 +1,146 @@
+// This optional code is used to register a service worker.
```
Comment (carsonfarmer) [inline src/serviceWorkerRegistration.ts:1]: this is how we register it, copied from cra

**Review by brunellaa — APPROVED** [review]

**Pull request merged**